spec/tapioca/cli/dsl_spec.rb

```diff
@@ -343,6 +343,66 @@ def title=(title); end
           assert_success_status(result)
         end
 
+        it "generates RBI files for engine when provided with an `app_root` flag" do
+          @project.write("lib/post.rb", <<~RB)
+            require "smart_properties"
+
+            class Post
+              include SmartProperties
+              property :title, accepts: String
+            end
+          RB
+
+          @project.write("lib/comment.rb", <<~RB)
+            require "smart_properties"
+
+            module Namespace
+              class Comment
+                include SmartProperties
+                property! :body, accepts: String
+              end
+            end
+          RB
+
+          engine_path = @project.path + "/test/dummy"
+
+          begin
+            FileUtils.mkdir_p(engine_path)
+            FileUtils.mv(@project.path + "/config", engine_path)
+            FileUtils.mv(@project.path + "/lib", engine_path)
```

Comment:
I'd prefer writing the files to `/test/dummy` directly on line 347 but not a blocker.

Reply:
Agreed, unfortunately the top-level `before(:all)` sets the project path so we would need to restructure the tests first.

Reply:
I'm a bit confused. I was only referring to the `lib` files. More specifically I was thinking of changing the 2 writes to:
`@project.write("test/dummy/lib/post.rb", <<~RB)` & `@project.write("test/dummy/lib/comment.rb", <<~RB)`

and removing the line `FileUtils.mv(@project.path + "/lib", engine_path)`.

We don't have to do it, I'm just trying to understand the project path blocker 🙂